pallets/gear/src/schedule.rs

```diff
@@ -792,8 +830,8 @@ impl<'a, T: Config> gas_metering::Rules for ScheduleRules<'a, T> {
         let max_params = self.schedule.limits.parameters;
 
         let weight = match *instruction {
-            End | Unreachable | Return | Else | Block(_) | Loop(_) | Nop | Drop => 0,
-            I32Const(_) | I64Const(_) => w.i64const,
+            End | Unreachable | Return | Else => 0,
```

Comment:
I had set 0 cost for loop, nop and etc instructions to zero, because they 100% removed in wasmer, so we shouldn's set weight for them.